nucliadb/nucliadb/common/cluster/base.py

```diff
@@ -109,11 +109,6 @@ async def delete_shard(self, id: str) -> str:
         resp: noderesources_pb2.ShardId = await self.writer.DeleteShard(req)  # type: ignore
         return resp.id
 
-    async def clean_and_upgrade_shard(self, id: str) -> noderesources_pb2.ShardCleaned:
-        req = noderesources_pb2.ShardId(id=id)
-        resp = await self.writer.CleanAndUpgradeShard(req)  # type: ignore
-        return resp
-
     async def del_vectorset(self, shard_id: str, vectorset: str) -> OpStatus:
```

Comment:
Should these ones be deleted too?

Reply:
I was going to split it in another PR to not mix cleans, but I can do it here too